spec/jobs/irs_attempts_events_batch_job_spec.rb

```diff
@@ -5,8 +5,8 @@
     context 'IRS attempts API is enabled' do
       let(:start_time) { Time.new(2020, 1, 1, 12, 0, 0, 'UTC') }
       let(:private_key) { OpenSSL::PKey::RSA.new(4096) }
-      let(:public_key) { private_key.public_key }
-      let(:encoded_public_key) { Base64.strict_encode64(public_key.to_der) }
+      #let(:public_key) { private_key.public_key }
+      let(:encoded_public_key) { Base64.strict_encode64(private_key.public_key.to_der) }
```

Comment:
This is sloppy for the moment, but I think we never used `public_key` and wanted to see if we could ditch it.